source/planner.html

```diff
@@ -15,130 +16,246 @@
     <div class="calendar">
       <div class="day">
         <header>Monday</header>
+
+
+        <script>
```

Comment:
we should put this in a separate file so its all in one place, consider moving it to planner.js.